CATesting/config.yaml

```diff
@@ -0,0 +1,119 @@
+# Constants
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 10}}
+{{$qps := DefaultParam .qps 10}}
+# Adjust the following two for your cluster -
+# kubectl get node -o json | jq -r ".items[].status.allocatable.cpu"
+{{$NODE_FREE_CPU := DefaultParam .NODE_FREE_CPU 1}}
+# kubectl get node -o json | jq -r ".items[].status.allocatable.memory"
+# In MB
+{{$NODE_FREE_MEM := DefaultParam .NODE_FREE_MEM 3840}}
+
+# Variables
+{{$podsPerNamespace := MultiplyInt $PODS_PER_NODE .Nodes}}
+{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
+# Pod Resource proportional to node to ensure consistent triggering of scale-up
+{{$POD_CPU_MILLI := DivideFloat $NODE_FREE_CPU (MultiplyFloat $PODS_PER_NODE .9)}}
+{{$POD_MEMORY_MB := DivideFloat $NODE_FREE_MEM (MultiplyFloat $PODS_PER_NODE .9)}}
+
+name: Simple_Bring_Up
```

Comment:
Some of the naming in this file is a little awkward and inconsistent. I'd recommend either lower-kabob-case or UpperCamelCase.

In the kubernetes world, it's typically as follows
1. YAML Keys use lowerCamelCase
2. YAML Values use UpperCamelCase for enum values (i.e. specifying a well known option as a proper noun)
3. YAML Values use lower-kabob-case for things like names that you create.

There are a ton of reasons why for this, which we can discuss at some point if you're interested.

Reply:
Sounds good.